kotlin/spring/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+server.port=3000
+server.error.whitelabel.enabled=false
```

Comment:
What is this for ?

Reply:
https://www.baeldung.com/spring-boot-custom-error-page

Maybe the accurate to the same for spring/spring webflux in java

Reply:
same as spring java